generation/WinSDK/enums.json

```diff
@@ -30632,5 +30669,136 @@
         "field": "dwFlags"
       }
     ]
+  },
+  {
+    "name": "BRUSH_STYLE",
+    "members": [
+      {
+        "name": "BS_SOLID",
+        "value": 0
+      },
+      {
+        "name": "BS_NULL",
+        "value": 1
+      },
+      {
+        "name": "BS_HOLLOW",
+        "value": "BS_NULL"
+      },
+      {
+        "name": "BS_HATCHED",
+        "value": 2
+      },
+      {
+        "name": "BS_PATTERN",
+        "value": 3
+      },
+      {
+        "name": "BS_INDEXED",
+        "value": 4
+      },
+      {
+        "name": "BS_DIBPATTERN",
+        "value": 5
+      },
+      {
+        "name": "BS_DIBPATTERNPT",
+        "value": 6
+      },
+      {
+        "name": "BS_PATTERN8X8",
+        "value": 7
+      },
+      {
+        "name": "BS_DIBPATTERN8X8",
+        "value": 8
+      },
+      {
+        "name": "BS_MONOPATTERN",
+        "value": 9
+      }
+    ],
+    "uses": [
+      {
+        "struct": "tagLOGBRUSH",
+        "field": "lbStyle"
+      },
+      {
+        "struct": "tagLOGBRUSH32",
+        "field": "lbStyle"
+      }
+    ]
+  },
+  {
+    "name": "TMPF_FLAGS",
+    "flags": "true",
+    "members": [
+      {
+        "name": "TMPF_FIXED_PITCH",
+        "value": 1
+      },
+      {
+        "name": "TMPF_VECTOR",
+        "value": 2
+      },
+      {
+        "name": "TMPF_DEVICE",
+        "value": 3
+      },
+      {
+        "name": "TMPF_TRUETYPE",
+        "value": 4
+      }
+    ],
+    "uses": [
+      {
+        "struct": "TEXTMETRICW",
+        "field": "tmPitchAndFamily"
```

Comment:
This field is only a byte, but this enum will be uint. You need to set the type of the enum to be a byte.